### PR TITLE
labeling over mean

### PR DESCRIPTION
**Pull request by ruifanp**
# Description
Labeling over mean, update after fixing my environment.

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?
Unit tests

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/source/implementations/labeling_excess_mean.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    data.columns = data.columns.droplevel(1)
+
+    # Get returns over mean numerically
+    numerical = excess_over_mean(data=data, lag=True)
+
+    # Get returns over mean as a categorical label
+    categorical = excess_over_mean(data=data, binary=True, lag=True)
+
+    # Get categorical forward looking monthly labels.
+    labels = excess_over_mean(data=data, binary=True, resample_by='M', lag=True)
```
Comment (PanPip) [inline docs/source/implementations/labeling_excess_mean.rst:99]: Please change from `excess_over_mean(data=data, ...` to `excess_over_mean(prices=data, ...` as this code will cause an error.

File: docs/source/implementations/labeling_excess_mean.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+.. automodule:: mlfinlab.labeling.excess_over_mean
+   :members:
+
+Example
+########
+Below is an example on how to create labels of excess over mean.
+
+.. code-block::
+
+    import pandas as pd
```
Comment (PanPip) [inline docs/source/implementations/labeling_excess_mean.rst:82]: `import pandas as pd` can be removed as it's unused

**Comment by PanPip** [comment]
Everything looks good. 🙂 

@Jackal08 ready to be approved.

**Comment by ruifanp** [comment]
reopen for merge @Jackal08

**Comment by Jackal08** [comment]
@PanPip will you please review and merge into dev?

**Pull request merged**